### PR TITLE
[WIP] Add otto file from @otto-schnurr.

### PR DESCRIPTION
**Pull request by otto-schnurr**
:warning: Still things to add.

# TODO

[ ] More content.

### CODE REVIEW

**Comment by hoolrory** [comment]
You're fired

File: otto.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Hello
+
+## Info
+- name: Otto
```
Comment (1419krusty) [inline otto.md]: Totally fake name :nail_care: :horse:

**Comment by EStallcup** [comment]
wow, I can't believe you call yourself a "dev"

File: otto.md (hunk 1 above)
Comment (otto-schnurr) [inline otto.md]: :+1:

**Comment by otto-schnurr** [comment]
omg

**Comment by githubteacher** [comment]
:heart_eyes:

**Comment by orderedlist** [comment]
yes

**Comment by orderedlist** [comment]
party time

**Pull request merged**